app/src/main/java/com/github/se/eventradar/model/User.kt

```diff
@@ -49,10 +51,10 @@ data class User(
   }
```

Comment:
toMap() function does not map the new "friendList" property.

Reply:
thanks you for all the comments, will make the changes now